pulp_rpm/app/viewsets.py

```diff
@@ -304,6 +620,72 @@ class RpmDistributionViewSet(DistributionViewSet):
     endpoint_name = "rpm"
     queryset = RpmDistribution.objects.all()
     serializer_class = RpmDistributionSerializer
+    queryset_filtering_required_permission = "rpm.view_rpmdistribution"
+
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": ["list"],
+                "principal": ["authenticated"],
+                "effect": "allow",
+                "condition": "has_model_perms:rpm.view_rpmdistribution",
+            },
+            {
+                "action": ["retrieve"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_model_or_obj_perms:rpm.view_rpmdistribution",
+            },
+            {
+                "action": ["create"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_model_perms:rpm.add_rpmdistribution",
+            },
+            {
+                "action": ["update", "partial_update"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_model_or_obj_perms:rpm.change_rpmdistribution",
```

Comment:
you should also consider permissions to view the repo/publication which is used to update the distribution